### ISSUE TITLE
Improve color contrast in contributor card in dark mode

### ISSUE THREAD

**Issue by yuanLeeMidori** [Seneca-CDOT/telescope#2049]
**What happened**:
![image](https://user-images.githubusercontent.com/50813726/112927123-7a12e780-90e2-11eb-840d-e57532b9bb11.png)
The pair of this background color and this font color doesn't pass AA rating.
**What should have happened**:
It should at least pass AA rating, or even better (AAA)
**How to reproduce it (as precise as possible)**:

**Anything else we need to know?**:

**Environment**:

- OS:
- Browser:
